package.json

```diff
@@ -34,64 +34,64 @@
   },
   "devDependencies": {
     "acorn-globals": "4.1.0",
-    "ajv": "5.2.2",
-    "ajv-keywords": "2.1.0",
+    "ajv": "5.5.2",
+    "ajv-keywords": "2.1.1",
     "ast-replace": "1.1.3",
     "atob": "2.0.3",
-    "autoprefixer": "6.7.0",
-    "ava": "0.15.2",
+    "autoprefixer": "7.2.3",
```

Comment:
I'll LGTM, but i'd like autoprefixer, cssnano, postcss, and postcss-import to be on a separate PR (these are all css processing tools that i'd like to manually test or create a golden file for)

Reply:
Gotcha. I'll hold off on merging this PR, and first do those upgrades in a separate PR. I'll ping this PR after that. Thanks for the comments!

Reply:
@erwinmombay Sent you #12512 with the CSS changes alone.

Reply:
Update: #12512 has now been tested with the golden file that was added in #12526, and should be merged soon.

Reply:
Merged and rebased. This PR no longer contains any CSS-related package changes.